client/src/components/App/App.jsx

```diff
@@ -135,10 +199,7 @@ function App() {
     }
 
     try {
-      const resp = await axios.post(
```

Comment:
Is there a reason we aren't waiting for the post response? 

Normally the post response would tell you if it was successful or not. If you fail to register on the backend we would like the front end to respond in some way.

Reply:
Or is it because of the try catch?